source/documentBase.py

```diff
@@ -48,6 +49,18 @@ class _TableSelection:
 	colSpan: int
 
 
+@dataclass
+class _TableCell:
+	"""
+	Contains information about current cell in the table.

```

Comment:
I think TableCell is too generic to always be the "current cell"

```suggestion
	Contains information about a cell in the table with matching tableID
```